panels/dev-info/ha-panel-dev-info.html

```diff
@@ -248,6 +248,21 @@ <h2>Log Details ([[selectedItem.level]])</h2>
     };
   }
 
+  ready() {
+    super.ready();
+    this.addEventListener('hass-service-called', ev => this.serviceCalled(ev));
+  }
+
+  serviceCalled(ev) {
+    // Check if this is for us
+    if (ev.detail.success && ev.detail.domain === 'system_log') {
+      // Do the right thing depending on service
+      if (ev.detail.service === 'clear') {
+        this._fetchData();
```

Comment:
Should we fetch data or just clear the local entries?

Reply:
Should be ok to just clear local entries, so I switched to doing so.